functions/Install-DbaFirstResponderKit.ps1

```diff
@@ -58,33 +48,47 @@ Install-DbaFirstResponderKit -SqlInstance server1\instance1 -Database master -Sq
 
 Logs into server1\instance1 with SQL authentication and then installs the FRK in the master database.
 
+.EXAMPLE 
+Install-DbaFirstResponderKit -SqlInstance sql2016\standardrtm, sql2016\sqlexpress, sql2014
+
+Logs into sql2016\standardrtm, sql2016\sqlexpress and sql2014 with Windows authentication and then installs the FRK in the master database.
+
+.EXAMPLE 
+$servers = "sql2016\standardrtm", "sql2016\sqlexpress", "sql2014"
+$servers | Install-DbaFirstResponderKit
+
+Logs into sql2016\standardrtm, sql2016\sqlexpress and sql2014 with Windows authentication and then installs the FRK in the master database.
+
 #>
 	
 	[CmdletBinding()]
 	param (
-		[Parameter(Mandatory = $True, ValueFromPipeline = $True)]
-		#[Alias("ServerInstance", "SqlInstance")]
-		[object]$SqlInstance,
-		[object]$Database,
-		[object]$SqlCredential
+		[Parameter(Mandatory, ValueFromPipeline)]
+		[Alias("ServerInstance", "SqlServer")]
```

Comment:
aliases are important for backwards compatibility. You may have had issues with them earlier because it should have been SqlServer and not SqlInstance